src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'book/ADD_BOOK';
+const DELETE_BOOK = 'book/DELETE_BOOK';
+
```

Comment:
- [ ] Kindly create actions using the _**duck pattern**_. For example, `const ADD_BOOK = 'bookstore/book/ADD_BOOK'`. You can read this [article](https://github.com/erikras/ducks-modular-redux) for more information. 👍